src/signalFactory.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+  import SignalProvider from './SignalProvider';
+
+export default function featureFlagged(mapSignalsToProps = signals => signals) {
```

Comment:
nit: I would name this function something else. it doesn't indicate that it is a factory to me. `createFeatureFlagged`... `provideSignals`...  even `featureFlag` sounds more like a verb indicating that the function is modifying the wrapped component.

clearly not a big deal, just worth a thought to make the API more descriptive of the intention